dev/release/js-source-release.sh

```diff
@@ -100,3 +142,61 @@ echo "Success! The release candidate is available here:"
 echo "  https://dist.apache.org/repos/dist/dev/arrow/${tagrc}"
 echo ""
 echo "Commit SHA1: ${release_hash}"
+echo ""
+echo "The following draft email has been created to send to the "
+echo "dev@arrow.apache.org mailing list"
+echo ""
+
+# Create the email template for the release candidate to be sent to the mailing lists.
+MESSAGE=$(cat <<__EOF__
+To: dev@arrow.apache.org
+Subject: [VOTE] Release Apache Arrow JS ${js_version} - RC${rc}
+
+Hello all,
+
+I\'d like to propose the following release candidate (rc${rc}) of Apache Arrow
+JavaScript version ${js_version}.
+
+The source release rc{$rc} is hosted at [1].
+
+This release candidate is based on commit
+${release_hash}
+
+Please download, verify checksums and signatures, run the unit tests, and vote
+on the release. The easiest way is to use the JavaScript-specific release
+verification script dev/release/js-verify-release-candidate.sh.
+
+The vote will be open for at least 24 hours and will close once
+enough PMCs have approved the release.
```

Comment:
We should change the time window in the template to the default "The vote will be open for at least 72 hours". The release manager can change this on an ad hoc basis if we wish to keep doing faster votes while things are moving quickly